akirataguchi115/lukuvinkkikirjasto
# Fix package name

## Description
I fixed the package name in the test class. The wrong package name caused the tests to fail.

## How to test:
Run `./gradlew test` and check that the build is successful and see if the checks have passed.
